### PR TITLE
skip backend monitor tests if no internet connection

### PR DESCRIPTION
**Pull request by nonhermitian**
### Summary
Makes it so that test suite does not require internet connection to pass.

### CODE REVIEW

**Comment by 1ucian0** [comment]
We used to pull from VCR to mock the network responses in those cases... is it not working?

**Comment by jaygambetta** [comment]
no it does not work.

File: test/python/tools/monitor/test_backend_monitor.py [hunk 1]
```diff
@@ -20,11 +20,15 @@
...
 from qiskit.test import QiskitTestCase, requires_qe_access
+from qiskit.util import _has_connection
+# Check if internet connection exists
+HAS_NET_CONNECTION = _has_connection('qiskit.org', 443)
 
 
 class TestBackendOverview(QiskitTestCase):
     """Tools test case."""
 
+    @unittest.skipIf(not HAS_NET_CONNECTION, "requries internet connection.")
```
Comment (ajavadia) [inline test/python/tools/monitor/test_backend_monitor.py]:
```suggestion
    @unittest.skipIf(not HAS_NET_CONNECTION, "requires internet connection.")
```

File: test/python/tools/monitor/test_backend_monitor.py [hunk 2]
```diff
@@ -38,6 +42,7 @@ def test_backend_overview(self, qe_token, qe_url):
         self.assertIn('Avg. T1:', stdout)
         self.assertIn('Num. Qubits:', stdout)
 
+    @unittest.skipIf(not HAS_NET_CONNECTION, "requries internet connection.")
```
Comment (ajavadia) [inline test/python/tools/monitor/test_backend_monitor.py]:
```suggestion
    @unittest.skipIf(not HAS_NET_CONNECTION, "requires internet connection.")
```

Comment (1ucian0) [reply test/python/tools/monitor/test_backend_monitor.py]: I fixed this typo in https://github.com/Qiskit/qiskit-terra/pull/2448

File: test/python/tools/monitor/test_backend_monitor.py (hunk 1 above)
Comment (1ucian0) [reply test/python/tools/monitor/test_backend_monitor.py]: I fixed this typo in https://github.com/Qiskit/qiskit-terra/pull/2448

**Pull request merged**